db/seeds/00_initial.js

```diff
@@ -0,0 +1,171 @@
+const knex = require('../../knexConnection.js');
+const axios = require('axios');
+var key = require('../../unsplashAccess.js');
+
+exports.seed = async (knex) => {
+
+  var names = ['2 A.m.',
```

Comment:
Instead of listing the first element of the array on same line as opening bracket, the best practice would be to list the first element on new line, atleast in my view :). (You may choose to leave this as is, if it is consistent with your approach in general)

Reply:
Done! This data was just copied over from Frankie's initial seeding script, but I agree that it's much cleaner the way you requested and how I'd normally do things myself.